app/api/webhook/github/route.ts

```diff
@@ -167,60 +167,23 @@ export const POST = async (req: Request) => {
 	if (comment && !comment.body.toLowerCase().includes('maige'))
 		return new Response('Irrelevant comment', {status: 202})
 
-	if (issue?.pull_request) {
-		const {
-			pull_request: {diff_url: diffUrl},
-			node_id: pullId
-		} = issue
-
-		// Get GitHub app instance access token
-		const app = new App({
-			appId: process.env.GITHUB_APP_ID || '',
-			privateKey: process.env.GITHUB_PRIVATE_KEY || ''
-		})
-
-		const octokit = await app.getInstallationOctokit(instanceId)
-
-		const response = await fetch(diffUrl)
-
-		if (!response.ok) return new Response('Could not fetch diff', {status: 503})
-
-		const data = await response.text()
-
-		await reviewer({
-			octokit: octokit,
-			input: `Instruction: ${comment?.body}\n\nPR Diff:\n${data}`,
-			pullId
-		})
-
-		return new Response('Reviewed PR', {status: 200})
-	}
-
 	if (
 		!(
 			(action === 'opened' && payload?.issue) ||
-			(action === 'created' && payload?.comment)
+			(action === 'created' && payload?.comment) ||
+			(action === 'opened' && payload?.pull_request)
```

Comment:
Does this condition prevents [the one below](https://github.com/RubricLab/maige/pull/55/files#diff-52032ba7797e13b0c74f7bc549029f17b98b3143853ec980000d84ce121e38b5R255)?

Reply:
maige inline code reply